src/MatrixAlgebraKit.jl

```diff
@@ -3,14 +3,16 @@ module MatrixAlgebraKit
 using LinearAlgebra: LinearAlgebra
 using LinearAlgebra: norm # TODO: eleminate if we use VectorInterface.jl?
 using LinearAlgebra: mul!, rmul!, lmul!, adjoint!, rdiv!, ldiv!
-using LinearAlgebra: sylvester
-using LinearAlgebra: isposdef, ishermitian, issymmetric
+using LinearAlgebra: sylvester, lu!
+using LinearAlgebra: isposdef, issymmetric, opnorm
```

Comment:
This can go again, it resulted from things I was trying with `PolarNewton`. 
```suggestion
using LinearAlgebra: sylvester
using LinearAlgebra: isposdef
```